data/categories/ha_hardware.yml

```diff
@@ -8408,6 +8416,15 @@
   - display_technology
   - measuring_tool_features
   - pattern
+- id: ha-15-36-39
+  name: Micrometers
```

Comment:
I think we need to add a type attribute as well for Outside Micrometer, Digital Micrometer, Inside Micrometer, Depth Micrometer, Indicating Micrometer, Bore Micrometer, Blade Micrometer, Screw Thread Micrometer, Disc Micrometer, Groove Micrometer

Reply:
Added! ✅